scripts/updateJavaPOMs.js

```diff
@@ -59,7 +83,11 @@ async function consolidatePomParent() {
   });
 }
 async function consolidatePomFlow() {
-  consolidate('pom-flow.xml', `${mod}/${name}-flow/pom.xml`)
+  if (proComponents.includes(componentName)){
```

Comment:
I'd rather have a function `isProComponent` that returns true by opening the LICENSE file and looking for some magic words

Reply:
so let us ignore this for now.